persistence/mongodb-record/src/test/scala/net/liftweb/mongodb/record/MongoFieldSpec.scala

```diff
@@ -553,6 +552,18 @@ object MongoFieldSpec extends Specification with MongoTestKit with AroundEach {
     }
   }
 
+  "MongoMapField" should {
+    "create itself from bson doc" in {
+      import scala.collection.JavaConversions._
+      val rec = MapTestRecord.createRecord
+      val map = Map("a" -> "4", "b" -> "5", "c" -> "6")
+      val doc = new Document(map)
+      rec.mandatoryStringMapField.setFromDocument(doc)
+      rec.mandatoryStringMapField.get must_== map
```

Comment:
Codacy complained about this `get` call here. It must think it's being called on an `Option`. Changing it to `value` should shut it up.